bin/hermes

```diff
@@ -812,28 +833,26 @@ def create_quest(args):
         logging.error("No hosts specified")
         return
 
-    response = request_get("/api/v1/eventtypes?limit=all")
-    event_types = response.json()["eventTypes"]
-
-    found_event_type = None
-    for event_type in event_types:
-        if (
-            event_type["category"] == args.category
-                and event_type["state"] == args.state
-        ):
-            found_event_type = event_type
+    try:
+        response = request_get("/api/v1/fates/{}".format(args.fate_id))
+        fate_json = response.json()
 
-    if found_event_type is None:
+    except Exception:
```

Comment:
I don't like seeing naked exceptions unless you really, truly don't care about what kind of error it is. The error message is indicating that there's no matching fate, but what if that's not actually the error? Two suggestions:

1. Either literally make it a naked except (`except:`) and make the error more generic ('Error while getting fate...')
2. Or, make the exception handling more specific (like `except Http400Error` as an example), and keep the error message specific, but then add and additional generic handler with the more generic error message.